src/read-appointments.ts

```diff
@@ -30,7 +30,8 @@ export async function readAppointments(
 
 export async function readS4AppointmentsByPerson(person: Person, year: number): Promise<TimeSheetEntry[]> {
   const personId = person.s4ID;
-  const from = moment.utc(`${year}-01-01`);
+  //the appointments are in 2020 but are changed to the current year in util/time-util.ts
+  const from = moment.utc(`2020-01-01`);
```

Comment:
But then we should be consistent. Either get rid of the year parameter or use it. Is there a reason the "to" variable was not considered here?

Reply:
The "from" is 2020 or 2019 because the appointments would not not be read otherwise. The "to" is current year in case someone adds a new appointment in the current year.

If we modify the mock server parser directly maybe we could maybe have better way of changing the year but I'm not sure it's even possible.

Reply:
I am having difficulties understanding this explanation. `from` is the beginning of the year and `to` is the end of the year. If the mockserver gives you dates outside this range, I don't think we should change those, we should only adjust locally created dates. Otherwise this is too complicated to be understood by someone who comes here to learn. (It distracts from the actual flow). Instead we should change the dates in the mockserver itself. There you could easily do this with a variable in the `-data.ja` files.